src/main/java/by/custom/utilcalculator/domain/tree/Node.java

```diff
@@ -1,14 +1,19 @@
 package by.custom.utilcalculator.domain.tree;
 
+import by.custom.utilcalculator.domain.constants.Command;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+
 import java.util.ArrayList;
 import java.util.List;
 
 public class Node {
-    private final Node parent;
+    private Node parent;
     private final List<Node> children;
     private final String key;
```

Comment:
It's better to use enum instead of String.
You can look [deserialization node example](https://github.com/MaximPilipeiko/jackson/blob/master/src/main/java/org/example/Main.java)

Reply:
Fixed it. I don't know what happened - but now it works.